### PR TITLE
feat(SPV-809): mysql & mongo removal

### PR DESCRIPTION
**Pull request by wregulski**
# Pull Request Checklist

- [x] 📖 I created my PR using provided  : [CODE_STANDARDS](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_STANDARDS.md)
- [x] 📖 I have read the short Code of Conduct: [CODE_OF_CONDUCT](https://github.com/bitcoin-sv/spv-wallet/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] 🏠 I tested my changes locally.
- [ ] ✅ I have provided tests for my changes.
- [x] 📝 I have used conventional commits.
- [x] 📗 I have updated any related documentation.
- [x] 💾 PR was issued based on the Github or Jira issue.

### CODE REVIEW

File: engine/go.mod
Comment (chris-4chain) [inline engine/go.mod:1]: I think, after this mongo removal - we could think how to simplify one thing and completely remove ("mongo") leftovers.
- For example, look here: https://github.com/bitcoin-sv/spv-wallet/blob/feat/remove-mongo-mysql/engine/model_metadata.go

Here, there are packages imported, as: `go.mongodb.org/mongo-driver/bson`, `go.mongodb.org/mongo-driver/bson/bsontype`.

Generally, the methods (in this file) for `Metadata` are necessary to properly serialize and deserialize key-value object.
However, after mongo-support removal, I'm not sure if all are necessary. 

I guess we could investigate it - but maybe in another task 🤔

**Review by chris-4chain — APPROVED** [review]
Nice job! 🚀 
One thing which I've mentioned, can be done in separate task, I guess 😄

File: engine/go.mod
Comment (wregulski) [reply engine/go.mod:1]: Yes, exactly. I wanted to propose it as a separate task. I was scared that it might affect Metadata.

Comment (wregulski) [reply engine/go.mod:1]: Created task: SPV-808

**Pull request merged**